hotham/src/systems/pointers.rs

```diff
@@ -54,7 +36,10 @@ pub fn pointers_system_inner(
     let global_from_stage = stage::get_global_from_stage(world);
 
     // Create a transform from local space to grip space.
-    let grip_from_local = ROTATION_OFFSET * POSITION_OFFSET;
+    // NOTE: This is most likely *WRONG* as the order for these transforms was not recoreded correctly.
+    // TODO: Make these correct.
+
+    let grip_from_local = Affine3A::from_rotation_translation(ROTATION_OFFSET, POSITION_OFFSET);
```

Comment:
I think you should be able to do something like
```suggestion
    let grip_from_local = Affine3A::from_rotation_translation(ROTATION_OFFSET, Vec3::ZERO) * 
        Affine3A::from_rotation_translation(Quat::IDENTITY, POSITION_OFFSET);
```

Reply:
Agh, rats, I forgot to update the comment! I ended up just asking `nalgebra` what the correct values _should_ be and updated them accordingly

Reply:
It may still make more sense to express them in a more natural order where the offset is along a single axis.

Reply:
It may even make more sense to express the offset as `local_from_grip` and then invert it.

Reply:
Good thinking! Happy to come back to this later though, it's not the end of the world